client/src/plugins/usage-statistics/event-handlers/PingEventHandler.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import BaseEventHandler from './BaseEventHandler';
+
+const TWENTY_FOUR_HOURS_MS = 1000 * 60 * 60 * 24;
+
+// Sends a ping event to ET when it is enabled for the first time
+// and once every 24 hours.
+export default class PingEventHandler extends BaseEventHandler {
+  constructor(params) {
+
+    const { onSend } = params;
+
+    super('ping', onSend);
+
+    this._intervalID = null;
+
+    this.sentInitially = false;
+  }
+
+  setInterval = (func) => {
+    setInterval(func, TWENTY_FOUR_HOURS_MS);
+  }
```

Comment:
This method returns `undefined` so we don't save interval ID.

I'd rather use `setTimeout` and just schedule another call when the function is called. It's easy to unintentionally leave `setInterval` running in the background.

Reply:
🐵

Reply:
Bad mistake

Reply:
Fixed via: https://github.com/camunda/camunda-modeler/pull/1840/commits/36f2a2c0c89904df1ec65bd04f6f69270c7d6703

Good catch!